doc/source/getting_started/running_mapdl.rst

```diff
@@ -138,7 +132,25 @@ Alternatively you can use a hostname:
 
 Note that you must have started MAPDL in gRPC mode on the computer with
 the mentioned IP address/hostname for this to work.
+
 If you have MAPDL installed on your local host, you
 can use the ``launch_mapdl`` method to both start and connect to MAPDL.
```

Comment:
I believe you wanted me to flag occurrences where the function directive wasn't used so that it was a link to the API reference topic.

Reply:
You are right. Well done.